float32 pressure
float32 temperature
float32 humidity
